### DISCUSSION TITLE
In Hugo, does the CMS require Netlify Identity Widget?

### DISCUSSION THREAD

**Discussion by sardarms** [sveltia/sveltia-cms#213] [Q&A] [answered]
Hi, I have a question. Do we really need to put the below code in the `<head>` section? 

```
<script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
``` 

I migrate from Decap CMS and Decap require this in Hugo (not sure about other ssg). But this slow down the site and hit the performace. Today I realised that this code may not rquire for Sveltia CMS as my sites seems working without it. If this is the case, this will be a very good improvement. Current documentation does not clearly say anything about it. 

Can you please take a look and if not require, update the documentation?

**Comment by kyoshino** [comment, marked as answer]
You don’t need the Netlify Identity widget script. It’s only needed if you’re using the Git Gateway backend. I have updated the [README](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#migrating-from-git-gateway-backend) to explain it. (Full documentation is coming later!)

**Comment by sardarms** [comment]
Thank you!. In the full documentation you can say that this will improve performance as no third party server and script will be looked for while loading the site.
